pkg/hints/ec_hint.go

```diff
@@ -0,0 +1,118 @@
+package hints
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/lambdaclass/cairo-vm.go/pkg/hints/hint_utils"
+	"github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
+	"github.com/lambdaclass/cairo-vm.go/pkg/types"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+)
+
+type BigInt3 struct {
+	Limbs []lambdaworks.Felt
+}
+
+type EcPoint struct {
+	X BigInt3
+	Y BigInt3
+}
+
+func (val *BigInt3) Pack86() big.Int {
+	sum := big.NewInt(0)
+	for i := 0; i < 3; i++ {
+		felt := val.Limbs[i]
+		signed := felt.ToSigned()
+		shifed := new(big.Int).Lsh(signed, uint(i*86))
+		sum.Add(sum, shifed)
+	}
+	return *sum
+}
+
+func FromBaseAddr(addr memory.Relocatable, virtual_machine vm.VirtualMachine) (BigInt3, error) {
```

Comment:
```suggestion
func BigInt3FromBaseAddr(addr memory.Relocatable, virtual_machine vm.VirtualMachine) (BigInt3, error) {
```
Name is not descriptive enough and will clash with other types as we usa a universal package for hints